.circleci/config.yml

```diff
@@ -1,3 +1,29 @@
+e2eTest: &e2eTest
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/apprclient
+    steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./e2e-harness
+
+    - run: ./e2e-harness localkube
+
+    - run: ./e2e-harness setup --remote=false
+
+    - run: ./e2e-harness test --test-dir=integration/test/basic
+
+    - run:
+        name: Finish with cleanup, no matter if the test succeeded or not
+        command: ./e2e-harness teardown
+        when: always
```

Comment:
So here I noticed we do not need the CNR server magic because the integration tests works against the fixures in the local FS. Is that about right?

Reply:
Yep, that's only needed for `apprclient`.